helm/generic-node/templates/watchdog.yaml

```diff
@@ -0,0 +1,72 @@
+---
+{{if .Values.watchdog.enabled}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Values.name}}-watchdog
+  labels:
+    app: {{.Values.name}}-watchdog
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{.Values.name}}-watchdog
+  template:
+    metadata:
+      labels:
+        app: {{.Values.name}}-watchdog
+        testnet: {{$.Values.testnetName}}
+        role: watchdog
+        version: {{trunc 6 (split ":" $.Values.watchdog.image)._1 | trimSuffix "-"}}
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: {{.Values.ports.metrics | quote}}
+        prometheus.io/path: '/metrics'
+    spec:
+      containers:
+      - name: watchdog
+        resources:
+          requests:
+            memory: {{.Values.watchdog.memory}}
+            cpu: {{.Values.watchdog.cpu}}
+        image: {{.Values.watchdog.image}}
+        command: ["python3"]
```

Comment:
You shouldnt have to overwrite the command and args here, as long as the defaults in the Dockerfile are correct